e2e/scenario/fixture/apps/formauth/authOnce.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE HTML>
```

Comment:
this file looks as subset of app.html, can't we use app.html directly ?.
If necessary you can extend it with the footer, this won't break existing tests